notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>It would be good to explain what science products are in comparison to auxiliary and noise (and anything from the list above that I missed).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>